git-extra/git-sdk.sh

```diff
@@ -45,23 +45,25 @@ sdk () {
 		echo "$*" >&2
 		return 1
 		;;
-	init)
+	init-worktree)
 		case "$2" in
 		build-extra|git|MINGW-packages|MSYS2-packages)
-			dest="/usr/src/$2"
-			if test ! -d "$dest"/.git
-			then
-				mkdir -p "$dest" &&
-				git init "$dest" &&
-				git -C "$dest" remote add origin https://github.com/git-for-windows/$2
-			fi &&
-			git -C "$dest" pull origin master
+			mkdir -p /usr/src/"$2" &&
+			git -C /usr/src/"$2" init &&
+			git -C /usr/src/"$2" config core.autocrlf false &&
+			git -C /usr/src/"$2" remote add origin \
+				https://github.com/git-for-windows/"$2" ||
+			sdk die "Could not initialize /usr/src/$2"
 			;;
 		*)
 			sdk die "Unhandled repository: $2" >&2
 			;;
 		esac
 		;;
+	init)
+		init_lazy "$2" &&
```

Comment:
I don't see `init_lazy` defined anywhere. Did you mean `sdk init-worktree`?

Reply:
Whoops! You're right, I wanted to rename init-worktree to init-lazy...